app/src/main/java/com/mapbox/vision/examples/utils/ContextEx.kt

```diff
@@ -4,4 +4,13 @@ import android.content.Context
 
 fun Context.dpToPx(dp: Float) = dp * this.resources.displayMetrics.density
 
-fun Context.pxToDp(px: Float) = px / this.resources.displayMetrics.density
\ No newline at end of file
+fun Context.pxToDp(px: Float) = px / this.resources.displayMetrics.density
+
+enum class Resource(val resourceType: kotlin.String) {
+    Drawable("drawable"),
+    String("string")
+}
+
+fun Context.getResId(name: String, type : Resource) = resources.getIdentifier(
```

Comment:
android already has system 'Resources' class. May be better to rename this to 'ResourceName' or 'ResourceType' to avoid confuse. What do you think?

Reply:
Ok, lets rename